test/utilities/update_dom_by_api_request_test.js

```diff
@@ -0,0 +1,86 @@
+require("../setup");
+const utilities = require('../../app/javascript/src/utilities.js');
+
+
+describe('utilities.updateDomByApiRequest', function() {
+  var targetId = "update-dom-by-api-request-target-element";
+  var insertId = "update-dom-by-api-request-insert-element";
+  var get;
+
+  before(function() {
+    get = $.get;
+    $(document.body).append("<p id=\"" + targetId + "\"></p>");
+    $.get = function(urlNotNeeded, response) {
+      response("<span id=\"" + insertId + "\">Luke</span>");
+    }
+  });
+
+  afterEach(function() {
+    $(document.body).find("#" + insertId).remove();
+  });
+
+  after(function() {
+    $(document.body).find("#" + targetId).remove();
+    $.get = get;
+  });
+
+  it('should place new element after target node', function() {
```

Comment:
Should this one be `it('should place new element inside target node')` or maybe `should append new element within target node`?  As the expectation is `expect($targetNode).find($insertedNode)` so we're looking for elements within `$targetNode`

Reply:
Yes, it should be inside/'append to' the target node as it is testing the default, single param, action. Have now corrected.